code/game/objects/effects/temporary_visuals.dm

```diff
@@ -96,3 +96,20 @@
 	splatter_type = "csplatter"
 	color = BLOOD_COLOR_SYNTHETIC
 
+//------------------------------------------
+//Shockwaves
+//------------------------------------------
+
+/obj/effect/temp_visual/shockwave
+	icon = 'icons/effects/light_overlays/shockwave.dmi'
+	icon_state = "shockwave"
+	plane = DISPLACEMENT_PLATE_RENDER_LAYER
+	pixel_x = -496
+	pixel_y = -496
+
+/obj/effect/temp_visual/shockwave/Initialize(mapload, radius)
+	. = ..()
+	deltimer(timerid)
```

Comment:
If you're gonna do that you might want to consider just not using a temp_visual subtype at all, it's the only thing it does :P